File: testWeb618/configs/entities/AppAccount.js
module.exports = {
    Name: "AppAccount",
    PrimaryKey: "AppAccountId",
    Properties: GetProperties(),
    StatusDataSource: GetStatusDataSource()
}

function GetProperties() {
    return [
        GetProperty("AppAccountId", "AppAccountId"),
        GetProperty("AccessPathName", "访问路径名"),
        GetProperty("CompanyName", "公司名称"),
        GetProperty("LogoImageUrl", "公司Logo"),
        GetProperty("Address", "地址"),
        GetProperty("Linkman", "联系人"),
        GetProperty("Phone", "手机"),
        GetProperty("AppId", "微信AppId"),
        GetProperty("Secret", "微信Secret"),
        GetProperty("DeveloperWeChat", "微信开发者微信号"),
        GetProperty("StatusName", "状态"),
        GetProperty("CreateDate", "创建时间")
    ]
}

function GetProperty(Name, Label) { return { Name, Label } }

function GetStatusDataSource() {
    return [{ Value: 1, Text: "正常" }, { Value: 2, Text: "关闭" }]
}

File: Marriage/MarriageApi/Marriage.Entity/Domain/AppAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Marriage.Entity.Domain
{
    /// <summary>
    /// App账号
    /// </summary>
    public class AppAccount
    {
        /// <summary> 
        /// 主键
        /// </summary> 
        public Guid AppAccountId { get; set; }
        /// <summary> 
        /// 后台访问路径名，建议用字母集合
        /// </summary> 
        public string AccessPathName { get; set; }
        /// <summary> 
        /// 微信AppId
        /// </summary> 
        public string AppId { get; set; }
        /// <summary> 
        /// 微信Secret
        /// </summary> 
        public string Secret { get; set; }
    }
}
